test/tests/meshgenerators/fancy_extruder_generator/tests

```diff
@@ -28,4 +28,14 @@
     design = 'FancyExtruderGenerator.md'
     issues = '#16902'
   []
+  [extruder_with_element_extra_interger_swap]
+    type = 'Exodiff'
+    input = fancy_extruder_with_element_extra_integer_swap.i
+    exodiff = 'fancy_extruder_with_element_extra_integer_swap_out.e'
+    recover = false
```

Comment:
```suggestion
```

I believe that `recover = false` is only needed when we're doing `--mesh-only`. Because you're actually executing something here (even with `solve = false`), this shouldn't be needed